qiskit_nature/second_q/hamiltonians/electronic_energy.py

```diff
@@ -60,7 +61,10 @@ def __init__(
             nuclear_repulsion_energy: the optional nuclear repulsion energy.
             reference_energy: an optional reference energy (such as the HF energy).
         """
-        super().__init__(self.__class__.__name__, electronic_integrals, shift=energy_shift)
+        self._electronic_integrals: dict[ElectronicBasis, dict[int, ElectronicIntegrals]] = {}
+        for integral in electronic_integrals:
+            self.add_electronic_integral(integral)
+        self._shift = energy_shift or {}
```

Comment:
Should we allow getter access to these shift values if users want to read them more directly? We expose these after interpret into the final result but we could provide them similarly via a getter here too. Is there a reason we have setters on these - do we ever want to change these after construction where I see they all default to None.

Reply:
The `_shift` will get refactored further once the `PolynomialTensor` is merged. Hence, I left it untouched for now :+1: